scripts/compare_benchmarks.py

```diff
@@ -337,8 +337,13 @@ def double_vertical_separators(lines, vertical_separators_to_duplicate):
     green_control_sequence = colored("", "green")[0:5]
     red_control_sequence = colored("", "red")[0:5]
 
-    table_string_reformatted = "```diff\n"
-    table_string_reformatted += create_context_overview(old_data, new_data, github_format) + "\n"
+    table_string_reformatted = "<details>\n" + \
+                               "<summary>Configuration Overview - click to expand</summary>\n\n" + \
+                               "```diff\n" + \
+                               create_context_overview(old_data, new_data, github_format) + \
+                               "```\n" + \
+                               "</details>\n\n" + \
+                               "```diff\n"
```

Comment:
The diff doesn't close within the closing `</details>`.

Reply:
What about line 344?

Reply:
Ahhh, got it. It’s the next diff...